.changeset/enhance-factory.md

```
@@ -0,0 +1,108 @@
+---
+"@daiso-tech/core": minor
+---
+
+Added new middleware utility `enhanceFactory` that allows apply middlewares to methods.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)